source/lib/JMSLab/builders/build_latex.py

```diff
@@ -44,3 +46,40 @@ def add_call_args(self):
                                   os.path.normpath(self.log_file))
         self.call_args = args
         return None
+     
+    def add_bib_name(self, target):
+        if bool(target):
+            target = misc.make_list_if_string(target)
+            target_file = str(target[0]).split(".")[0]
+        else:
+            target_file = ''
+        self.bib_file = target_file
+        return None
+    
+    def do_call(self):
+        '''
+        Acutally execute the system call attribute.
+        Raise an informative exception on error.
+        '''
+        self.bibtex_executable  = 'bibtex'
```

Comment:
@mcaceresb, if `biblatex` is used, then should the command be `biber`, and do you think this is something that we should try to allow for?

Reply:
@veli-m-andirin I think it would be useful. However, I don't think we should be too smart about it. It would be sufficient to allow the user to specify a BibTeX executable (like any of the other executables).

Reply:
@veli-m-andirin @mcaceresb In fa4dad5 I added some functionality so that `self.bibtex_executable` is based on the `executable.yaml` and so can be changed just like any of the other executables if someone would prefer to use `biblatex` or `biber`.

Reply:
Thanks @rcalvo12, this implementation looks good to me. @mcaceresb, please feel free to resolve if it looks good to you, too.